Compilers.sln

```diff
@@ -438,15 +438,13 @@ Global
 		{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C} = {A41D1B99-F489-4C43-BBDF-96D61B19A6B9}
 		{9508F118-F62E-4C16-A6F4-7C3B56E166AD} = {E35DA3D1-16C0-4318-9187-6B664F12A870}
 		{F5CE416E-B906-41D2-80B9-0078E887A3F6} = {E35DA3D1-16C0-4318-9187-6B664F12A870}
-		{A41D1B99-F489-4C43-BBDF-96D61B19A6B9} = {32A48625-F0AD-419D-828B-A50BDABA38EA}
```

Comment:
>{A41D1B99-F489-4C43-BBDF-96D61B19A6B9} = {32A48625-F0AD-419D-828B-A50BDABA38EA} [](start = 2, length = 79)

Was changing the SLN file intentional?

Reply:
Looks like you're changing hte folder structure here. Don't think this should be a part of the change.

---
In reply to: [220336965](https://github.com/dotnet/roslyn/pull/30063#discussion_r220336965) [](ancestors = 220336965)